.eslintrc.json

```diff
@@ -568,7 +568,9 @@
 					"vscode-oniguruma",
 					"iconv-lite-umd",
 					"tas-client-umd",
-					"jschardet"
+					"jschardet",
```

Comment:
Maybe you have already discussed this in the API Sync, but FYI that the timeline api uses timestamp https://github.com/microsoft/vscode/blob/a4d426a1c2ece4222f1343459d31fe948098b2f6/src/vscode-dts/vscode.proposed.timeline.d.ts#L14, not sure if we wanna keep it aligned.